static/src/javascripts/boot-webpack.js

```diff
@@ -1,3 +1,39 @@
-import boot from 'bootstraps/standard/main';
+import domready from 'domready';
+import raven from 'common/utils/raven';
+import bootStandard from 'bootstraps/standard/main';
+import config from 'common/utils/config';
 
-boot();
+// let webpack know where to get files from
+// eslint-disable-next-line camelcase,no-undef
+__webpack_public_path__ = `${config.page.assetsPath}javascripts/`;
```

Comment:
Asked Simon in person about this. Would it be worth adding a comment explaining why this variable is in this format?